pom.xml

```diff
@@ -22,7 +20,6 @@
   <name>MathWorks Polyspace Plugin</name>
   <url>https://github.com/jenkinsci/mathworks-polyspace-plugin</url>
   <packaging>hpi</packaging>
-  <description>Jenkins plugin for Polyspace</description>
```

Comment:
Descriptions have been inlined into index.jelly and will cause build failures soon.